packages/lib/src/components/internal/SecuredFields/lib/CSF/partials/processAutoComplete.ts

```diff
@@ -12,7 +12,7 @@ import getIframeContentWin from '../utils/iframes/getIframeContentWin';
  *
  * @param pFeedbackObj -
  */
-export function processAutoComplete({ csfState, csfConfig, csfCallbacks }, pFeedbackObj: SFFeedbackObj): void {
+export function processAutoComplete({ csfState, csfConfig, csfCallbacks }, pFeedbackObj: SFFeedbackObj): boolean {
```

Comment:
Maybe rename the function as well, such as `shouldProcessAutoComplete` or similar? Because the returned value is `boolean` now, the function name doesn't reflect it.

Reply:
Returning a boolean is less to do with whether we process the event or not, and more to do with returning something that a unit test (still to be written) can inspect

Reply:
OK, then leave it for now :). 
It's more about giving function a name that describes what is does and align with the returned type. So if the return type is void, the function name is more like 'do' something. If the returned type is a boolean, the function name is more like 'should', 'is'.